queries/src/test/scala/io/shiftleft/queries/MallocMemcpyTests.scala

```diff
@@ -30,7 +30,7 @@ class MallocMemcpyTests extends WordSpec with Matchers {
       * */
     "find calls to malloc/memcpy system with different expressions in arguments" in {
 
-      val src = cpg.call("malloc").filter(_.argument(1).arithmetics).l
+      val src = cpg.call("malloc").filter(_.argument(1)).l
```

Comment:
I'm not sure about this one: wasn't the point of this query to look for arithmetic operations in first arguments to malloc?

Reply:
Added a commit adding that back.